modules/sdk/src/main/scala/org/tessellation/sdk/domain/transaction/TransactionValidator.scala

```diff
@@ -22,17 +21,16 @@ trait TransactionValidator[F[_], T <: Transaction] {
 }
 
 object TransactionValidator {
+  import eu.timepit.refined.auto._
+  val stardustPrimary: Address = Address("DAGSTARDUSTCOLLECTIVEHZOIPHXZUBFGNXWJETZVSPAPAHMLXS")
+  val lockedAddresses: Set[Address] = Set(
+    stardustPrimary
+  )
```

Comment:
@ryle-goehausen At Alex's request we removed the other two locked addresses and added this one. Would be good to confirm that is correct.

Reply:
Glad I checked. I'll put them back.